packages/account/src/providers/provider.ts

```diff
@@ -451,6 +451,29 @@ export default class Provider {
     }
   }
 
+  private static handleBasicAuth(url: string): [string, ProviderOptions['requestMiddleware']] {
```

Comment:
Rename to fit next suggestions.

```suggestion
  private static extractBasicAuth(url: string): [string, ProviderOptions['requestMiddleware']] {
```